docs/administrators/operational_procedures.rst

```diff
@@ -33,23 +33,31 @@ Context
 Procedure
 ++++++++++
 
-1. Verify in `project.yml` that `install_mode` is set to `release`.
-2. Freeze all modules with
+1. Verify in ``project.yml`` that ``install_mode`` is set to ``release``.
+2. Freeze all modules with:
 
 .. code-block:: bash
 
-    inmanta -vv -X project freeze --recursive --operator "=="
+    inmanta -vv project freeze --recursive --operator "=="
```

Comment:
Considering how bad we sometimes are at error messages, perhaps we should keep the `-X`.

Reply:
I missed the fact that the `-X` option was part of the top command, adding it back